rerun.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh -ex
```

Comment:
I assume you didn't want to commit this file?

Reply:
Lol nope, will fix